### ISSUE TITLE
ImageField Not Rendering Image

### ISSUE THREAD

**Issue by jeffbernst** [marmelab/react-admin#4497]
**What you were expecting:**

An `ImageInput` and `ImageField` should render the same image if passed the same `label` and `source`. 

**What happened instead:**

This is the code in the component that is leading to the error:

```js
      {disabled
        ? <ImageField
          label="Image Url"
          source={getSource('imageUrl')} />
        : <ImageInput
          label="Image Url"
          source={getSource('imageUrl')} />
      }
```

The `ImageInput` renders as expected, and by inspecting the component with devtools you can see it is wrapping an `ImageField` that is displaying the image. The `ImageField` just renders an empty `div`.

**Steps to reproduce:**

Add the above code to a component with the react-admin components imported.

**Other information:**

**Environment**

* React-admin version: 3.0.1
* Last version that did not exhibit the issue (if applicable): N/A
* React version: 16.12.0
* Browser: Chrome
* Stack trace (in case of a JS error): N/A

**Comment by nathanschepers** [comment]
I think I'm running into the same problem. I see you closed this without comment. If you resolved it, how?

**Comment by jeffbernst** [comment]
Sorry it's been awhile since I worked on this and I can't remember exactly what ended up fixing the issue. I should've left a comment for future reference!

I found the PR where I was working on this though, and it looks like the code that I ended up with was:
```js
<ImageField
  label="Image Url"
  record={scopedFormData}
  source="imageUrl"
/>
```
So for some reason I didn't call `getSource()` on the source, and then I also passed `scopedFormData` into the `record`. Not sure if this will fix it for you, but good luck!
